lib/erb_lint/formatters/json_formatter.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+module ERBLint
+  module Formatters
+    class JSONFormatter
+      RUBOCOP_MESSAGE_PATTERN = /\A(?<linter>\S+): (?<message>.+)\z/.freeze
+
+      def initialize
+        @result = {
+          metadata: metadata_hash,
+          files: [],
+          summary: {offense_count: 0, inspected_file_count: 0}
+        }
+      end
+
+      def file_completed(relative_filename, runner)
+        @result[:summary][:offense_count] += runner.offenses.count
+        @result[:summary][:inspected_file_count] += 1
+        @result[:files] = hash_for_file(relative_filename, runner)
+      end
+
+      def report(stats, options, io)
+        io.puts JSON.pretty_generate(@result)
+      end
+
+      private
+
+      def metadata_hash
+        {
+          erb_lint_version: ERBLint::VERSION,
+          ruby_engine: RUBY_ENGINE,
+          ruby_version: RUBY_VERSION,
+          ruby_patchlevel: RUBY_PATCHLEVEL.to_s,
+          ruby_platform: RUBY_PLATFORM
+        }
+      end
+
+      def hash_for_file(relative_filename, runner)
+        {
+          path: relative_filename,
+          offenses: runner.offenses.map { |o| hash_for_offense(o) }
+        }
+      end
+
+      def hash_for_offense(offense)
+        linter, message = linter_and_message(offense)
+        {
+          message: message,
+          linter: linter,
```

Comment:
For now, if a rubocop cop found an offense, this is set to the particular cop name and not to `'rubocop'` which is the actual erb-lint-linter. I first thought this is better but now think, maybe it is Ok/better to have `'rubocop'` here and let the cop remain in the message. Thoughts on this?

Reply:
I am agree with you, it would be nice to identify from where ()